python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -66,7 +66,8 @@ def __init__(self, uri):
         try:
             self.connection = psycopg2.connect(expandedConnInfo)
         except self.connection_error_types() as e:
-            err = str(e)
+            # get credentials if cached or assking to the user no more than 3 times
+            err = unicode(e)
```

Comment:
`str()` is preferable for python3 (and I think this targets master?)

Reply:
right!